tests/foreman/cli/test_oscap.py

```diff
@@ -637,9 +615,12 @@ def test_positive_associate_scap_policy_with_hostgroup_via_ansible(self):
         assert scap_policy['deployment-option'] == 'ansible'
         assert scap_policy['hostgroups'][0] == hostgroup['name']
 
+    @pytest.mark.parametrize('deploy', **parametrized(['manual', 'puppet', 'ansible']))
```

Comment:
Also, Make sure you add the `:parameterized: yes` token to the test docstring.

Reply:
What does deploy meaning here? Can we rename to some better opt ?

Reply:
deploy here means using which way we want to deploy scap. This feels ok and matches with hammer option as well.